src/include/duckdb/main/capi/header_generation/apis/v1/v1.2/v1.2.0.json

```diff
@@ -288,6 +288,7 @@
         "duckdb_appender_column_type",
         "duckdb_appender_error",
         "duckdb_appender_flush",
+        "duckdb_appender_clear",
```

Comment:
Instead of adding the function to the stable API, we need to first expose it only in the unstable part. Have a look at `src/include/duckdb/main/capi/header_generation/README.md` for adding functions to the unstable part, or have a look at the folder containing examples: `src/include/duckdb/main/capi/header_generation/apis/v1/unstable`.